spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -943,2259 +807,1399 @@ header, and the mode is OUT.
+In the “wrapped” operation style, the input and output messages contain
+a single part which refers (through the _element_ attribute) to a global
+element declaration (the _wrapper_)
 
-| a|
-maps to a header.
+of complexType defined using the xsd:sequence compositor. The global
+element declaration for the input message has a local name equal to
+@WebMethod.operationName. The global element declaration for the output
+message (if it exists) has a local name equal to
+@WebMethod.operationName + “Response”. Both global element declarations
+appear in the @WebService.targetNamespace.
 
-If the target namespace is set to “”, this represents the empty
+Non-header method parameters and return values map to child elements of
+the global element declarations defined for the method. The order of
+parameters in the parameter list determines the order in which the
+equivalent child elements appear in the operation’s global element
+declarations.
 
-a|
-return value does not map to a header,
+The @WebParam.name and @WebParam.targetNamespace annotation elements
+determine the QName of a parameter’s child element, while the
+@WebResult.name and @WebResult.targetNamespace annotations determines
+the QName of the return value’s child element. The schema type for each
+child element is derived from the type of the Java parameter or return
+value, as described in section _5 Java Mapping To XML/WSDL._
 
-Otherwise, the default is
+==== Document “Bare” Style
 
-| a|
-namespace.
+In the “bare” operation style, the input and output messages contain a
+single part which refers (through the _element_ attribute) to an element
+that is mapped from the method parameter and return value. The QName of
+the input body element is determined by the values of the @WebParam.name
+and @WebParam.targetNamespace annotations on the method parameter, and
+the QName of the output body element is determined by the values of the
+@WebResult.name and @WebResult.targetNamespace annotations. The schema
+types for the input and output body elements are derived from the types
+of the Java parameter or return values, as described in section _5 Java
+Mapping To XML/WSDL._
 
-a|
-the targetNamespace for the Web Service.
+Web Services that use the document “bare” style MUST adhere to the
+following restrictions:
 
-a|
-header
+* If the operation is marked @Oneway, it MUST have a void return value, a
+single non-header parameter marked as IN, and zero or more header
+parameters.
+* If the operation is not marked @Oneway, it may have one of the following
+forms:
 
-a|
-If true, the parameter is in the message header rather then the message
-body.
+* A non-header parameter marked as IN, a non-header parameter marked as
+OUT, a void return value, and zero or more header parameters.
+* A single non-header parameters marked as IN_OUT, a void return value,
+and zero or more header parameters.
+* A single non-header parameter marked as IN, non-void return value and
+zero or more header parameters.
```

Comment:
this has to be nested by one more level as all these items belong to the "not marked @Oneway" point

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/62fdf397f28a148826367da36bcaef2c30e97c7d